modules/karpenter/variables.tf

```diff
@@ -17,92 +17,101 @@ variable "cluster_name" {
 }
 
 ################################################################################
-# IAM Role for Service Account (IRSA)
+# Pod Identity IAM Role
```

Comment:
It does not appear this is the case but the naming comes across as confusing. IRSA and Pod Identity are two distinct forms of authentication, is the intent here that module consumers should switch to Pod Identity or is IRSA still the direction?

Reply:
Correct, Pod Identity is the direction

Reply:
If that is the case there are other references to IRSA in the Karpenter Module that may need to be addressed? I am assuming this is just the first step since the Assume Role Policy still appears to be IRSA Compatible. Is there a timeline on when consumers would have to additionally support EKS Pod Identity to continue using this component of the module?